R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -167,6 +167,18 @@ test_that("spark.lapply should perform simple transforms", {
   sparkR.session.stop()
 })
 
+test_that("add jar should work and allow usage of the jar on the driver node", {
+  sparkR.sparkContext()
+
+  destDir <- paste0(tempdir(), "/", "testjar")
```

Comment:
you may want to use https://stat.ethz.ch/R-manual/R-devel/library/base/html/file.path.html instead for windows compatibility